core/src/main/java/discord4j/core/object/util/PermissionSet.java

```diff
@@ -87,54 +98,126 @@ private PermissionSet(final long rawValue) {
     }
 
     /**
-     * Performs a logical AND of of this permission set with the other permission set.
+     * Performs a logical <b>AND</b> of this permission set with the other permission set.
+     * <p>
+     * The resultant set is the <b>intersection</b> of this set and the other set. A permission is contained iff it was
```

Comment:
Should probably write out if-and-only-if, not everyone knows this abbreviation.

Reply:
And this is `Set#retainAll`

Reply:
"iff" can be found in the JDK, but sure.